tests/UI/campaigns/functional/BO/08_design/01_themeAndLogo/03_advancedCustomization/01_advancedCustomization.js

```diff
@@ -0,0 +1,277 @@
+    it('should login go to \'Design > Theme & Logo\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToThemeAndLogoPage_1', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.designParentLink,
+        dashboardPage.themeAndLogoParentLink,
+      );
+
+      await themeAndLogoPage.closeSfToolBar(page);
+
+      const pageTitle = await themeAndLogoPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(themeAndLogoPage.pageTitle);
+    });
+
+    it('should go to \'Advanced Customization\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAdvancedCustomizationPage_1', baseContext);
+
+      await themeAndLogoPage.goToSubTabAdvancedCustomization(page);
+
+      const pageTitle = await advancedCustomizationPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(advancedCustomizationPage.pageTitle);
+    });
+
+    it('should download theme', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'downloadTheme', baseContext);
+
+      filePath = await advancedCustomizationPage.downloadTheme(page);
+      const exist = await files.doesFileExist(filePath);
+      await expect(exist, 'Theme was not downloaded').to.be.true;
+    });
+
+    it('should click on upload child theme button', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnUploadChildTheme', baseContext);
+
+      const modalTitle = await advancedCustomizationPage.clickOnUploadChildThemeButton(page);
+      await expect(modalTitle, 'Modal \'Upload child theme\' is not displayed')
+        .to.contains('Drop your child theme archive here or select file');
+    });
+
+    it('should upload child theme', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'uploadChildTheme', baseContext);
+
+      await files.renameFile(filePath, renamedFilePath);
+      const uploadTheme = await advancedCustomizationPage.uploadTheme(page, renamedFilePath);
+      await expect(uploadTheme, 'Child theme is not uploaded')
+        .to.contains('The child theme has been added successfully.');
+    });
+
+    it('should close the modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'closeModal', baseContext);
+
+      const isModalClosed = await advancedCustomizationPage.closeModal(page);
+      await expect(isModalClosed, 'Modal not closed').to.be.true;
+    });
+
+    // remove the child_classic.zip from themes folder
+    it('should remove the zip file of the child theme from the themes folder', async function () {
+      await testContext
+        .addContextItem(this, 'testIdentifier', 'removeZipFileChildThemeFromThemesFolder', baseContext);
+
+      const generatedFilePath = await files.getFilePathAutomaticallyGenerated(themesPath, renamedFilePath);
+      await files.deleteFile(generatedFilePath);
```

Comment:
Maybe you should add an expect to verify if the file is deleted successfully
```suggestion
      await files.deleteFile(generatedFilePath);
      const doesFileExist = await files.doesFileExist(generatedFilePath);
      await expect(doesFileExist).to.be.false;
```